app/src/operations/mail/index.ts

```diff
@@ -14,45 +14,76 @@ export default defineOperationApp({
 			label: '$t:operations.mail.to',
 			text: Array.isArray(to) ? to.join(', ') : to,
 		},
+		{
+			label: '$t:type',
+			text: type,
+		},
 		{
 			label: '$t:operations.mail.body',
 			text: body,
 		},
 	],
-	options: [
-		{
-			field: 'to',
-			name: '$t:operations.mail.to',
-			type: 'csv',
-			meta: {
-				width: 'full',
-				interface: 'tags',
-				options: {
-					placeholder: '$t:operations.mail.to_placeholder',
-					iconRight: 'alternate_email',
+	options: (panel) => {
+		return [
+			{
+				field: 'to',
+				name: '$t:operations.mail.to',
+				type: 'csv',
+				meta: {
+					width: 'full',
+					interface: 'tags',
+					options: {
+						placeholder: '$t:operations.mail.to_placeholder',
+						iconRight: 'alternate_email',
+					},
 				},
 			},
-		},
-		{
-			field: 'subject',
-			name: '$t:subject',
-			type: 'string',
-			meta: {
-				width: 'full',
-				interface: 'input',
-				options: {
-					iconRight: 'title',
+			{
+				field: 'subject',
+				name: '$t:subject',
+				type: 'string',
+				meta: {
+					width: 'full',
+					interface: 'input',
+					options: {
+						iconRight: 'title',
+					},
 				},
 			},
-		},
-		{
-			field: 'body',
-			name: '$t:operations.mail.body',
-			type: 'text',
-			meta: {
-				width: 'full',
-				interface: 'input-rich-text-md',
+			{
+				field: 'type',
+				name: '$t:type',
+				type: 'string',
+				schema: {
+					default_value: 'input-rich-text-md',
+				},
+				meta: {
+					interface: 'select-dropdown',
+					width: 'full',
```

Comment:
Let's do `half` width for now and circle back if `full` ended up being preferred 👍

Reply:
https://github.com/directus/directus/pull/15514/commits/1d905d85c5db6d1a9f1e80abf4d43fb1a0aa025f#diff-cae570b585c5b54e12ca50e49754c44ef14dd7e06249343d7f309e634aade5ffR61-R63